app/templates/request-common-confirm-send-by-post.html

```diff
@@ -77,6 +77,12 @@
     {%- set question_title = _('Do you want to send a new household access code to this address?') -%}
 {%- endif -%}
 
+{%- if (display_region == 'ni') -%}
+    {%- set large_print = 'Large print – suitable for visually impaired' -%}
```

Comment:
Can you change the variable name to large_print_legend?

Reply:
Done.